### DISCUSSION TITLE
Unrecognized SKIP comment...  that's weird!

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#186] [Q&A] [answered]
Well I can delete those "unrecognized" comments... if it makes the SKIP transpiler a bit happier...

But a comment is a comment ... there is no Skip comment style!

Hey that's my LAST issue... I've got a building App... fire up the devices - it's testing time!

<img width="522" alt="Screenshot 2024-08-13 at 6 17 05 PM" src="https://github.com/user-attachments/assets/acc72cb8-e346-44b7-851a-9a9fc4e2d93b">

**Comment by marcprux** [comment, marked as answer]
> Well I can delete those "unrecognized" comments... if it makes the SKIP transpiler a bit happier...
> 
> But a comment is a comment ... there is no Skip comment style!

Skip supports various annotations in comments, like "SKIP ATTRIBUTES" and "SKIP REPLACE", as described at https://skip.tools/docs/platformcustomization/#skip-comments. We interpret your `// SKIP does not support` comment as an attempt to use a Skip-specific annotation, and are warning you that we don't support the "does not support" annotation.

The quick fix is to just lower-case "SKIP" to "Skip" there, or else just ignore the warning.

> Hey that's my LAST issue... I've got a building App... fire up the devices - it's testing time!

Hooray! Let us know how it goes!

**Reply by davidakoontz** [reply]
WOW!  How-about-that!  I'm Wrong again...

I love it when **wrong** turns into more knowledge!

Thanks for all the hand holding Marc.... maybe I counted my chickens too quickly...

Does NOT Compile <  Compiles < Does NOT Run < Runs on my machine

I forgot about getting the AppState object into the App via iOS Environment.  I got it compiling - but NOT running.
